pepper-apis/src/main/java/org/broadinstitute/ddp/pex/TreeWalkInterpreter.java

```diff
@@ -97,7 +104,7 @@ public boolean eval(String expression, Handle handle, String userGuid, String ac
         }
 
         InterpreterContext ictx = new InterpreterContext(handle, userGuid, activityInstanceGuid);
-        PexValueVisitor visitor = new PexValueVisitor(this, ictx);
+        PexValueVisitor visitor = new PexValueVisitor(this, ictx, activityInstanceSummary);
```

Comment:
I think it would have been easier if we made `UserActivityInstanceSummary` a part of `InterpreterContext`, after all it does feel like a piece of "context" data.